lib/client-assets.php

```diff
@@ -404,6 +407,17 @@ function gutenberg_register_scripts_and_styles() {
 			'wp-url',
 			'wp-viewport',
 		),
+		filemtime( gutenberg_dir_path() . 'build/block-library/index.js' ),
+		true
+	);
+	// Remove it with 3.8.0 release.
```

Comment:
Can we tag this so it's findable or have an issue for it referenced here? Anything so this doesn't get lost.

Reply:
I think it's fine, I think we don't need the comment at all because the deprecated call on the frontend will be caught and when we remove it, the module will be empty.

Reply:
Ah, okay, gotcha! That makes sense, I'll ignore it then. 👍